x-pack/plugins/ml/public/application/components/import_export_jobs/import_jobs_flyout/import_jobs_flyout.tsx

```diff
@@ -45,9 +45,10 @@ import { useEnabledFeatures } from '../../../contexts/ml';
 
 export interface Props {
   isDisabled: boolean;
+  refreshJobs: (() => void) | null;
```

Comment:
I suggest renaming this callback, e.g. `onImportComplete`, because it's not necessarily meant to be for job list refresh. 

```suggestion
  onImportComplete: (() => void) | null;
```

Reply:
Updated in f55c97f22f14e3996b40610143d0413c7ae1a6df